github-orgs/kubeflow/org.yaml

```diff
@@ -1010,6 +1010,22 @@ orgs:
             privacy: closed
             repos:
               katib: write
+          wg-data-leads:
+            description: Team of Model Registry and Spark Operator
+            members:
+            - andreyvelich
+            - ckadner
+            - dhirajsb
+            - isinyaaa
+            - lampajr
+            - rareddy
+            - rkubis
+            - tarilabs
+            - Tomcli
+            - tonyxrmdavidson
```

Comment:
Are all of them required to establish this WG? 

1. If anyone of them has not joined Kubeflow yet, it will break our pipeline. Follow https://github.com/kubeflow/internal-acls?tab=readme-ov-file#joining-kubeflow-github-organization to ask them to join kubeflow org first, or remove them from this list for now.
2. Some of them seem not needed for model registry project, please only keep the owners of this project here.

Reply:
removed, only members of KF are listed now. As other folks join the org will submit another PR to add them